relation_engine_server/test/test_pull_spec.py

```diff
@@ -0,0 +1,37 @@
+import unittest
+from unittest import mock
+import re
+
+from relation_engine_server.utils.pull_spec import download_specs
+from relation_engine_server.utils.wait_for import wait_for_api
+from spec.test.test_ensure_specs import ensure_borked_indexes
+
+
+class Test(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        wait_for_api()
+
+    def test_download_specs(self):
+        download_specs(init_collections=True, reset=True)
```

Comment:
what's actually being tested here?

Reply:
It's ascertaining that with the real specs and the spec initialization, the new appended ensure code in `download_specs` does NOT fail. It's pretty much redundant bc `download_specs` is called (eventually in the stack) in the `setUpClass` in other test classes with most probably the real specs and the spec initialization, but it's not expensive and it kind of rounds out this test class ...

Reply:
Will leave documentation?

Reply:
Or is it crufty

Reply:
is there any kind of output or something that you can test somewhere?

Reply:
Yes, done